.gitignore

```diff
@@ -73,3 +73,8 @@ typings/
 
 # Storybook
 storybook-static/
+
+# IDE files
+# VSCode does not read local project files that might execute binaries by
+# default due to security so this config wouldn't have an effect, anyway.
+.vscode
```

Comment:
Why do we need this if no `.vscode` file is being added? Or does the procedure added to the docs create one? I didn't see any changes to a file in `.vscode` locally, but I did already have a file there.

Reply:
To fix the local TS problem, one needs to create a `.vscode` directory but this shouldn't ever get checked in, which is why I ignored it.

Reply:
Do you really? I thought is was persisted via localstorage.

Reply:
I'm not sure what you mean. Is what persisted in local storage?

Reply:
My understanding from reading the issue is that when you use the UI in VSCode to indicate you want to use the project's version of TS, that choice is stored in localstorage, so I'm not sure what the file in `.vscode` is for.

Reply:
When I used the UI to make the choice, VSCode saved the configuration to a local `.vscode/settings.json` file. The [security issue](https://github.com/microsoft/vscode/issues/30069#issuecomment-312732928) was saying that local storage is used to know if a user has interacted with the UI rather than opened some project with an existing `.vscode/settings.json` file, I think.

Reply:
I would argue that this folder should be ignored in your global gitignore: https://help.github.com/en/github/using-git/ignoring-files#create-a-global-gitignore 🙂